sui/src/api.rs

```diff
@@ -128,6 +128,30 @@ pub trait RpcGateway {
         digest: TransactionDigest,
     ) -> RpcResult<TransactionEffectsResponse>;
 
+    #[method(name = "getTransactionsByInputObject")]
+    async fn get_transactions_by_input_object(
+        &self,
+        object: ObjectID,
+    ) -> RpcResult<Vec<(GatewayTxSeqNumber, TransactionDigest)>>;
```

Comment:
Whats the rational for returning the seq number and not just the transaction hash?

Also do we want to enforce some limit on the number of responses via the request parameters? like the last 20 txns (of course now that I bring this up i now understand the need for the seq number)

Reply:
I'm following the convention of the other methods here that return txes, and I think the reason you point out is correct.